ui/ui-config/pages/app/new/index.tsx

```diff
@@ -4,7 +4,8 @@ import { useState, useEffect } from "react";
 import { Form, Modal, Spinner, Alert } from "react-bootstrap";
 import { useRouter } from "next/router";
 import AppSelectorList from "@grouparoo/ui-components/components/AppSelectorList";
-import { ErrorHandler } from "@grouparoo/ui-components/utils/errorHandler";
+import { errorHandler } from "@grouparoo/ui-components/eventHandlers";
+import { ErrorHandler } from "@grouparoo/ui-components/eventHandlers/errorHandler";
```

Comment:
I don't think you need the type any more

Reply:
Removed!